content/docs/demos/cert-manager_integration.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Cert-manager Certificate Provider"
+description: "Using cert-manager as a certificate provider"
+type: docs
+weight: 30
+---
+
+This guide demonstrates the usage of [cert-manager][1] as a certificate provider to manage and issue certificates in OSM.
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes v1.19.0 or greater.
+- Have OSM installed.
```

Comment:
You have a note below stating that cert-manager must be installed and running before osm is installed to use cert-manager as its certificate provider. Maybe remove this from the pre-requisites ?